src/main/kotlin/app/model/LocalRepo.kt

```diff
@@ -3,9 +3,21 @@
 
 package app.model
 
+import org.eclipse.jgit.lib.Config
+
 data class LocalRepo(var path: String = "") {
```

Comment:
Yep, data keyword means that equals, hashCode and toString methods will be auto-generated according to variables in primary constructor.